lib/train-habitat/illegal_state_error.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class IllegalStateError < StandardError
+  def initialize(msg = nil)
+    super
+  end
+end
+
+class NoServicesFoundError < IllegalStateError
+  def initialize(origin, name)
+    super("Expected one service '#{origin}/#{name}', but found none.")
+  end
+end
+
+class MultipleservicesFoundError < IllegalStateError
```

Comment:
I'd like to see Services capitalized, ie. `MultipleServicesFoundError`

Reply:
Oops that was a typo.